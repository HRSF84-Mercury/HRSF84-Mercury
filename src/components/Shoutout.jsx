import React from 'react';

const Shoutout = ({ shoutout }) => {
  return (
    <div className="shoutout list-item container column">
      <div className="">{shoutout.text}</div>
      <div className="">{shoutout.name}</div>
      <div className="">{shoutout.category}</div>
    </div>
  )
};

export default Shoutout;

