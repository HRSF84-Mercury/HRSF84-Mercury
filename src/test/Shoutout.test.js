import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Shoutout from '../components/Shoutout';

describe('Shoutout', () => {
  it('should render correctly with props', () => {
    const shoutout = {
      'name':'Some Awesome Shoutout',
      'text':'Some amazing shoutout text',
      'category':'Meme'
    };
    const output = shallow(
      <Shoutout shoutout={shoutout} />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});

