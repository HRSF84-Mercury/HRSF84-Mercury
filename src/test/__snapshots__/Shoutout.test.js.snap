// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shoutout should render correctly with props 1`] = `
<div
  className="shoutout list-item container column"
>
  <div
    className=""
  >
    Some amazing shoutout text
  </div>
  <div
    className=""
  >
    Some Awesome Shoutout
  </div>
  <div
    className=""
  >
    Meme
  </div>
</div>
`;
